script/Arcueid/Day 12/12_00_ARC12_2B.txt

```diff
@@ -365,50 +365,51 @@ My dalliance with the body was little more than a passing fancy. Nothing came of
 [sha:38f28cbea70c5f041d4b11911cb23da312f9b0dd]{
 -- Page 99, Offset 30823.
 -- そんな不確かな儀式は魔術とは呼べない。それは呪いと言うのだ、ベスティーノ」
-Such a dubious rite cannot be called magecraft. It is merely a curse, Bestino."
+Magecraft isn't the right word for such a dubious rite. 'Curse' is the better descriptor, Bestino."
 }
 [sha:e8b4d6570c70e9f6901c07a1f96199de3e9540e9]{
 -- Page 99, Offset 30824.
 -- 「“…………つまり？”」
-"...In other words..."
+"You're saying...?"
 }
 [sha:3bdb58b6f22d0ed6bf7c672ce28d43376a0b45b1]{
 -- Page 100, Offset 30825.
 -- 「徒労だったな。不老の解決法はない。せいぜい健やかに、<死|・><ぬ|・><ま|・><で|・><苦|・><し|・><め|・>」
-"It was a waste of effort.
- There's no way to obtain eternal youth.
- The best you can do is suffer out the time before you die as vigorously as you're able."
+//せいぜい here isn't the usual meaning, it's part of an adverbial expression involving the 命令形. Basically, he's actually adressing Mario here.
+"I'm saying it was a waste of time.
+ The puzzle of 'how not to age' cannot be solved.
+ I hope suffering to death will prove invigorating."
```

Comment:
Would add a 'for you' to the end of the third line. It almost sounds like he's talking about himself at the moment.